package.json

```diff
@@ -37,15 +36,7 @@
     "@rollup/plugin-json": "^5.0.2",
     "@rollup/plugin-node-resolve": "^13.1.1",
     "@rollup/plugin-typescript": "^8.3.0",
-    "@storybook/addon-actions": "^6.5.9",
-    "@storybook/addon-docs": "^6.5.9",
-    "@storybook/addon-essentials": "^6.5.9",
-    "@storybook/addon-interactions": "^6.5.9",
-    "@storybook/addon-links": "^6.5.9",
-    "@storybook/builder-webpack5": "^6.5.9",
-    "@storybook/manager-webpack5": "^6.5.9",
-    "@storybook/react": "^6.5.9",
-    "@storybook/testing-library": "^0.0.13",
+    "@testing-library/dom": "^9.3.3",
```

Comment:
@vojtechszocs FYI yarn was complaining about this missing peer dependency (being requested by `@testing-library/user-event` from one of the other `@testing-library` libs below).

Reply:
AFAIK, we only used React testing library due to lib-utils having React components for which we had unit tests.

I'm not sure we still need it, will check.